docs/en/revocation-lists.rst

```diff
@@ -212,142 +231,220 @@ The following HTTP Status Codes MUST be supported:
+- The Credential Issuer MUST verify that it is the legitimate Issuer of the Digital Credential to which each Status Assertion Request object refers.
 
+**Step 3 (Check for validity)**: The Credential Issuer checks that the User's attributes are not updated by the Authentic Source or that the latter has not revoked them. The technical mechanisms for obtaining this information are out-of-scope of this technical implementation profile. 
 
-**Step 4 (Status Attestation Creation)**: The Issuer creates the corresponding Status Attestation. A non-normative example of a Status Attestation is given below.
+**Step 4 (Status Assertion Creation)**: The Credential Issuer creates the corresponding Status Assertion. When a Status Assertion is requested to a Credential Issuer, the Credential Issuer checks the status of the Digital Credential and creates a Status Assertion bound to it. If the Digital Credential is valid, the Credential Issuer creates a new Status Assertion, which a non-normative example is given below where the format is JWT.
 
 .. code::
 
     {
-        "alg": "ES256",
-        "typ": "status-attestation+jwt,
-        "kid": $ISSUER-JWKID
-    }
-    .
-    {
-        "iss": "https://pid-provider.example.org",
-        "iat": 1504699136,
-        "exp": 1504700136,
-        "credential_hash": $CREDENTIAL-HASH,
-        "credential_hash_alg": "sha-256",
-        "cnf": {
-            "jwk": $CREDENTIAL-CNF-JWK
-            }
-    }
-
-**Step 4 (Status Attestation Response)**: The Issuer then returns the Status Attestation to the Wallet Instance, as in the following non-normative example.
+    "alg": "ES256",
+    "typ": "status-assertion+jwt",
+    "kid": $ISSUER-JWKID
+	}
+	.
+	{
+		"iss": "https://issuer.example.org",
+		"iat": 1504699136,
+		"exp": 1504785536,
+		"credential_hash": $CREDENTIAL-HASH,
+		"credential_hash_alg": "sha-256",
+		"credential_status_validity": true,
+		"cnf": {
+			"jwk": {...}
+		}
+	}
+
+**Step 4 (Status Assertion Response)**: The response MUST include a JSON object with a member named `status_assertion_responses`, which contains the Status Assertions and or the Status Assertion Errors related to the request made by the Wallet Instance, as in the following non-normative example.
 
 .. code::
 
-    HTTP/1.1 201 Created
-    Content-Type: application/json
-    
-    {
-        "status_attestation": "eyJhbGciOiJFUzI1NiIsInR5cCI6IndhbGxldC1...",
-    }
+    HTTP/1.1 200 Created
+	Content-Type: application/json
+
+	{
+		"status_assertion_responses": ["${base64url(json({typ: status-assertion+jwt, ...}))}.payload.signature", ... ]
+	}
+
+The member `status_assertion_responses` MUST be an array of strings, where each of them represent a Status Assertion Response object as defined in `[OAuth Status Attestation draft 01] <https://datatracker.ietf.org/doc/draft-demarco-status-attestations/01/>`_.
```

Comment:
```suggestion
The member `status_assertion_responses` MUST be an array of strings, where each of them represent a Status Assertion Response object as defined in `[OAuth Status Attestation draft 01] <https://datatracker.ietf.org/doc/draft-demarco-oauth-status-assertions>`_.
```


instead of duplicating the URL as a constant, it is required to create a ref to a defined resource, as made with all tyhe other specs